graph/cassandra/create_quadstore.go

```diff
@@ -0,0 +1,137 @@
+// Copyright 2014 The Cayley Authors. All rights reserved.
+//
+// Licensed under the Apache License, Version 2.0 (the "License");
+// you may not use this file except in compliance with the License.
+// You may obtain a copy of the License at
+//
+//     http://www.apache.org/licenses/LICENSE-2.0
+//
+// Unless required by applicable law or agreed to in writing, software
+// distributed under the License is distributed on an "AS IS" BASIS,
+// WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+// See the License for the specific language governing permissions and
+// limitations under the License.
+
+package cassandra
+
+import (
+	"github.com/barakmich/glog"
+	"github.com/gocql/gocql"
+
+	"github.com/google/cayley/graph"
+)
+
+func createNewCassandraGraph(addr string, options graph.Options) error {
+	cluster, err := clusterWithOptions(addr, options)
+	if err != nil {
+		return err
+	}
+	cluster.Consistency = gocql.All
+	session, err := cluster.CreateSession()
+	if err != nil {
+		glog.Errorln("Could not create a Cassandra graph:", err)
+		return err
+	}
+	err = session.Query(`
+	CREATE TABLE log (
```

Comment:
this will default to size tiered compaction which would be bad for ttl'ing out data. It might be nice to have this as DateTieredCompactionStrategy with a reasonable max_size so that after n days if all the keys are expired C* just has to rm the SSTable vs having to do a compaction operation on it. For log type data that's a natural fit for a time-series type model.